CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to CICE
+
+Welcome to the CICE Consortium ! First, thanks a lot for considering to contribute to CICE.
+
+There are many ways to contribute, and we appreciate your efforts. For an introductory tour, head over to our [Contribution guide][contributing] and take a look at the [Resources page], our go-to index with links to code, documentation, testing information, guidelines, and more.
+
+[contributing]: https://github.com/CICE-Consortium/About-Us/wiki/Contributing
+[resources]: https://github.com/CICE-Consortium/About-Us/wiki/Resource-Index
+
+## Getting help
+If you still need help getting started using the model afer reviewing the [model documentation][doc-running], the first point of contact with the CICE Consortium is the [Consortium Community Forum][forum]. 
+This forum is monitored by Consortium members and also opened to the whole community.
+**Please do not use our issue tracker for general support questions.**
+
+[doc-running]: https://cice-consortium-cice.readthedocs.io/en/master/user_guide/ug_running.html
```

Comment:
As indicated below by @duvivier it might be good to point to the documentation on the resource page rather than directly into the readthedocs.  As with everything, I recommend we point to the resource page as much as we can, then have the resource page point out to the actual docs.

Reply:
done